jekyll/_docs/npm-continuous-deployment.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: classic-docs
+title: "Continuous deployment with npm"
+short-title: "npm publish from CircleCI"
```

Comment:
The "short title" here isn't really shorter. I think we can just leave it off completely.